GUIDE_SORMAS2SORMAS_CERTIFICATE.md

```diff
@@ -5,6 +5,7 @@ This guide explains how to:
  * set up the server address list file
  * add certificates of other SORMAS instances to the local truststore
  * add other servers to the local server list
+ * add the ssl certificate of the other server into the key store of the local server
```

Comment:
This is about the CA store and not the key store.
Also, only on test systems the certificate should be added to cacerts.
This must not be done on production systems , because then this certificate is also trusted as CA.
For production systems certificates signed by valid CAs are required.

Reply:
Maybe `* handling self-signed ssl certificates on test systems` instead?